### ISSUE TITLE
Unsupported syntax causes TLAPM to crash

### ISSUE THREAD

**Issue by cipher1024** [tlaplus/tlapm#30]
When using the following syntax `[ f EXCEPT ![1,2] = 1 ]` (i.e. updating a function that takes multiple arguments) TLAPM crashes. I have attached a screenshot of the error message. It would be nice to be told that this syntax is not supported (if this can't be implemented easily) and be suggested a workaround (`[ f EXCEPT ![<<1,2>>] = 1 ]` perhaps). 

![image](https://user-images.githubusercontent.com/71008/106397296-a3f9a780-63da-11eb-84f8-91efd60b183c.png)

Here is a minimal working example:

```tla
-------------------------------- MODULE bar --------------------------------

VARIABLES f

MyDef == [ f EXCEPT ![1,2] = 1 ]

=============================================================================
```

**Comment by johnyf** [comment]
Thank you for reporting this, the relevant code in the parser is:

https://github.com/tlaplus/tlapm/blob/02b7f5df10155c85844eccb16088a4338c3ac775/src/expr/e_parser.ml#L380

which can parse a single expression between the square brackets, but not a sequence of expressions that are separated by commas. The `use (expr b)` could be replaced by a

```ocaml
choice [
    use (expr b);
    (sep (punct ",") (use (expr b))) <$> (fun es -> Tuple es)
]
```

similarly to parsing of tuple constructors:

https://github.com/tlaplus/tlapm/blob/02b7f5df10155c85844eccb16088a4338c3ac775/src/expr/e_parser.ml#L407

**Comment by cipher1024** [comment]
Nice thanks! Should I make the change?

**Comment by johnyf** [comment]
Actually, a few more changes were needed to address this area of syntax. I have opened #34 for addressing this issue.

**Comment by johnyf** [comment]
This issue has been addressed as of 5958dfaca70b07613603baadff57926aff7eba66.
